inc/shortcodes.php

```diff
@@ -29,7 +29,11 @@ function wp_contributions_plugin_card_shortcode( $atts ) {
 	), $atts ); // $atts['slug'] aka [wp_contributions_plugin_card slug="your-plugin-slug"]
 
 	wp_contributions_plugin_card( $args['slug'] );
+	$out = ob_get_contents();
```

Comment:
Rather than assigning the object buffer to the $out variable, using `ob_end_clean()` and returning it, instead you should just return `ob_get_clean()`